go.mod

```diff
@@ -47,7 +47,7 @@ replace (
 	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.18.6
 	k8s.io/kubectl => k8s.io/kubectl v0.18.6
 	k8s.io/kubelet => k8s.io/kubelet v0.18.6
-	k8s.io/kubernetes => k8s.io/kubernetes v1.18.6
+	k8s.io/kubernetes => ../../../k8s.io/kubernetes
```

Comment:
This points to a local kubernetes repository (with changes implemented [here](https://github.com/kubernetes/kubernetes/pull/93243#pullrequestreview-464074662))